lib/order.rb

```diff
@@ -0,0 +1,43 @@
+class Order
+
+def initialize
+  @list_of_dishes = {"pizza"=>10, "pasta"=>8, "garlic bread"=>5}
+  @order = {}
+end
+
+def display_menu
+  p @list_of_dishes
+end
+
+def add_to_order(item)
+  item.each do |key, value|
+  @order[key] = value
+  end
+end
+
+def display_order
+  p @order
+  p @order.length
+end
```

Comment:
Methods are really concise and tidy. Perhaps the order class is doing a bit too much - a Menu class to deal with display_menu and your dishes, having order deal with adding and checking out